### ISSUE TITLE
N/A being listed for created_at for snapshot during ls

### ISSUE THREAD

**Issue by shabazpatel** [datmo/datmo#193]

**Comment by shabazpatel** [comment]
```
bash-3.2$ datmo snapshot ls
+------------------------------------------+------------+---------------------------------------+-----------------------------------------+----------------------------+-------+
|                    id                    | created at |                 config                |                  stats                  |          message           | label |
+------------------------------------------+------------+---------------------------------------+-----------------------------------------+----------------------------+-------+
| 1460f06120701e250a5d7e55debb4a77c527e4bf |    N/A     |       {u'colsample_bytree': 0.8,      | {u'train accuracy': 0.9958333333333333, | creating my first snapshot |       |
|                                          |            | u'min_child_weight': 5, u'subsample': |    u'test accuracy': 0.95, u'title':    |                            |       |
|                                          |            |   0.8, u'eta': 0.1, u'objective': u   |   u'final model with specific params'}  |                            |       |
|                                          |            |     'binary-logistic', u'seed': 0,    |                                         |                            |       |
|                                          |            |            u'max_depth': 5}           |                                         |                            |       |
| 19ca895a8594b91dfaee57642f5a7e7c4be46626 |    N/A     |       {u'colsample_bytree': 0.8,      | {u'train accuracy': 0.9958333333333333, | creating my first snapshot |       |
|                                          |            | u'min_child_weight': 5, u'subsample': |    u'test accuracy': 0.95, u'title':    |                            |       |
|                                          |            |   0.8, u'eta': 0.1, u'objective': u   |   u'final model with specific params'}  |                            |       |
|                                          |            |     'binary-logistic', u'seed': 0,    |                                         |                            |       |
|                                          |            |            u'max_depth': 5}           |                                         |                            |       |
+------------------------------------------+------------+---------------------------------------+-----------------------------------------+----------------------------+-------+
```

**Comment by asampat3090** [comment]
fixed in e1c7eee
